.pipelines/restore-acquiring_code.cmd

```diff
@@ -0,0 +1,89 @@
+@echo off
+
+REM Prerequisities
+echo.
+echo Deleteing python from path...
+set PYTHONPATH=
+set PYTHONHOME=
+
+echo.
+echo Installing Visual Stuio 2019 workloads...
+c:
+mkdir c:\Downloads
+cd c:\Downloads
+curl -o vs_enterprise.exe -L "https://aka.ms/vs/16/release/vs_enterprise.exe"
+call C:\Downloads\vs_enterprise.exe modify --installPath "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise" --add Microsoft.VisualStudio.Component.VC.ATLMFC;includeRecommended --quiet --wait
```

Comment:
I think we need more workloads than just ATL.
Please add:
--add Microsoft.VisualStudio.Workload.NativeDesktop ^
--add Microsoft.VisualStudio.Workload.Universal

Reply:
These workloads are installed in the base image.

Reply:
ok